cli/generate_operator_keys.go

```diff
@@ -2,7 +2,10 @@ package cli
 
 import (
 	"encoding/base64"
+	"encoding/json"
+	"github.com/bloxapp/eth2-key-manager/encryptor/keystorev4"
```

Comment:
let's use `github.com/wealdtech/go-eth2-wallet-encryptor-keystorev4` instead because it still gets updated